token-lending/program/src/processor.rs

```diff
@@ -1994,6 +2022,115 @@ fn process_withdraw_obligation_collateral_and_redeem_reserve_liquidity(
     )
 }
 
+#[allow(clippy::too_many_arguments)]
+#[inline(never)] // avoid stack frame limit
+fn process_update_reserve_config(
+    program_id: &Pubkey,
+    optimal_utilization_rate: u8,
+    loan_to_value_ratio: u8,
+    liquidation_bonus: u8,
+    liquidation_threshold: u8,
+    min_borrow_rate: u8,
+    optimal_borrow_rate: u8,
+    max_borrow_rate: u8,
+    borrow_fee_wad: u64,
+    flash_loan_fee_wad: u64,
+    host_fee_percentage: u8,
```

Comment:
can we not do this with passing a config: ReserveConfig the same was `process_init_reserve` works?

Reply:
this would allow the validations that are right below to not be copy pasted and instead just have a `validate_reserve_configs` function perhaps?

Reply:
That seems pretty reasonable